templates/ci/xcode.yml

```diff
@@ -6,10 +6,14 @@
 pool:
   vmImage: 'macOS 10.13'
 
+variables:
+  Configuration: 'Release'
+  SDK: 'iphoneos'
+
 steps:
 - task: Xcode@5
   inputs:
+    actions: 'build'
+    xcWorkspacePath: '**/*.xcodeproj/project.xcworkspace'
     scheme: ''
-    sdk: 'iphoneos'
-    configuration: 'Release'
     xcodeVersion: 'default' # Options: 8, 9, default, specifyPath
```

Comment:
Do these inputs on the task (`sdk` and `configuration`) not have to be specified if the `SDK` and `Configuration` variables are defined above?

Reply:
They default to `$(SDK)` and `$(Configuration)`. But I went ahead and removed the variables since their utility probably wouldn't be obvious to a newcomer.